iotdb-cli/src/main/java/org/apache/iotdb/cli/client/AbstractClient.java

```diff
@@ -341,6 +347,44 @@ protected static Options createOptions() {
     return options;
   }
 
+  public static String getInstantWithPrecision(DateTimeFormatter formatter,
+                                               long timestamp, String timestampPrecision, ZoneId zoneid){
+
+    if(timestampPrecision.equals("us")){
```

Comment:
You can enumerate "ms", "us", "ns" in an enum class, rather than hard coded here.